sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -341,9 +402,9 @@ public void serialize(
             ClientSideRequestStatistics statistics, JsonGenerator generator, SerializerProvider provider) throws
             IOException {
             generator.writeStartObject();
-            long requestLatency = statistics
-                .getDuration()
-                .toMillis();
+            Duration duration = statistics
+                .getDuration();
+            long requestLatency = duration != null ? duration.toMillis() : 0;
```

Comment:
not related to this PR, but I wonder we should also use double for requestLatency. 

We are actually losing some precision by using .toMillis here. (the total request latency calculated by using the transport timeline could be slightly larger than the one calculated here -- found out during the benchmark diagnostic analysis)

Reply:
metrics don't allow floating point numbers (for histogram calculation). So we would need to determine whihc millisecond frament like millis in decimal * 100 or nanos etc. But since we just moved from nanos to ms I rather have some rounding errors - for metrics I think that is fine.

Thoughts?

Reply:
yea, I think it is fine